chemprop/models/mpn.py

```diff
@@ -60,8 +55,7 @@ def __init__(self, args: TrainArgs, atom_fdim: int, bond_fdim: int):
         self.W_o = nn.Linear(self.atom_fdim + self.hidden_size, self.hidden_size)
 
     def forward(self,
-                mol_graph: BatchMolGraph,
-                features_batch: List[np.ndarray] = None) -> torch.FloatTensor:
+                mol_graph: BatchMolGraph) -> torch.FloatTensor:
```

Comment:
I really like these changes! It's so much cleaner having all the features stuff outside of the MPN.